nishimweelysee/Elysee-CapstoneProject
# Ft user landing page

> #### What does this PR do?
>
>- i complete user landing page in this branch
>
> #### Description of Task to be completed
>
>- first page of website (index.html)
>- In content there is my Profession
>- Its linked with other pages that are not yet built
>- there is a css file for styling
>- there a js file for responsiveness
>- there is a font awesome styles from icons
>- hidden user profile image you can see on screenshoot.
> #### How should this be manually tested?
>- Open my webpage the first page you will see is my landing page
> #### Any background context you want to provide?
> the user profile icon is hidden it will be shown when user signed in with valid username and password
> #### What are the relevant Trello story?
> FT(user landing page) https://trello.com/b/TKrvwje3
> #### Screenshots (if appropriate)
> 
> - This shows a user profile image icon .<img width="909" alt="imgprofile" src="https://user-images.githubusercontent.com/56052588/92319506-43c42c80-f019-11ea-89d1-f767f0ca7f1b.PNG">
> - This is how the landing page look 
![image](https://user-images.githubusercontent.com/56052588/92319523-7c640600-f019-11ea-9dc4-754c89ec486d.png)

>Thank you!!
